cmd/kube-controller-manager/app/apps.go

```diff
@@ -40,6 +42,7 @@ func startDaemonSetController(ctx ControllerContext) (bool, error) {
 		ctx.InformerFactory.Core().V1().Pods(),
 		ctx.InformerFactory.Core().V1().Nodes(),
 		ctx.ClientBuilder.ClientOrDie("daemon-set-controller"),
+		flowcontrol.NewBackOff(5*time.Second, 15*time.Minute),
```

Comment:
what was the reasoning behind starting the backoff at 5s instead of something lower like 1s?

Reply:
The failures this backoff is used for are of more permanent nature. From those know to me this is often misconfiguration that needs manual intervention. I don't feels strong about it if you want to start at 1s.

Reply:
I'd start at 1s to stay more responsive for the first couple deletes.